pkg/asset/cluster/tfvars.go

```diff
@@ -34,6 +35,7 @@ func (t *TerraformVariables) Name() string {
 func (t *TerraformVariables) Dependencies() []asset.Asset {
 	return []asset.Asset{
 		&installconfig.InstallConfig{},
+		new(rhcos.Image),
```

Comment:
Why is this not `&rhcos.Image{}` like its neighbors?

Reply:
https://play.golang.org/p/NUhwmraqAtM
```console
prog.go:10:7: invalid pointer type *Image for composite literal
```

Reply:
https://golang.org/ref/spec#Address_operators